### DISCUSSION TITLE
Support large number of errors 🤡

### DISCUSSION THREAD

**Discussion by omega** [JetBrains/Qodana#69] [Ideas]
<img width="611" alt="image" src="https://user-images.githubusercontent.com/23787/152010567-2fbc4c63-c3c4-4ba8-be84-17275f33a5c9.png">

We have a code-base with a lot of warnings, and the UI breaks somewhat 😅

**Comment by brichbash** [comment]
Qodana UI should support more errors than that. In what environment do you open the report? 
Is there any chance you could share the report itself?

**Reply by omega** [reply]
I'm opening it in Firefox, if I set `white-space: nowrap` on the tab-counts, it works. I can't really share the whole report publically, but I could email you a link I suppose?

**Reply by brichbash** [reply]
Please send it to qodana-support@jetbrains.com

**Reply by brichbash** [reply]
Created an issue on [our bug tracker](https://youtrack.jetbrains.com/issue/QD-2372). If you have a spare minute, please follow it to get updated on it.
